packages/teko/tests/unit_tests/tDiagonallyScaledPreconditioner.cpp

```diff
@@ -132,7 +132,8 @@ const RCP<Thyra::LinearOpBase<ST> > buildSystem(const Teuchos::RCP<const Teuchos
    RCP<Tpetra::CrsMatrix<ST,LO,GO,NT> > mat = Tpetra::createCrsMatrix<ST,LO,GO,NT>(map,3);
 
    ST values[] = { -1.0, 2.0, -1.0};
-   GO iTemp[] = {-1,0,1}, indices[3];
+   int iTemp[] = {-1,0,1};
```

Comment:
I would need a Teko developer to tell me whether this test would make sense with `GO=unsigned long`.

Reply:
I agree - had the same concern.  If we're OK with some Teko tests failing if ```Tpetra_INST_INT_UNSIGNED_LONG:BOOL=ON```, I'd probably vote for reverting the logic to how it was before.

Reply:
@eric-c-cyr is OK with this change - in the unit tests, iTemp will never be so large that ``long`` will be needed.  Therefore keeping change.